pom.xml

```diff
@@ -77,9 +83,9 @@
 	
 	<dependencies>
```

Comment:
I'm on the fence. In other projects that have had global dependencies, I've left them in the parent. Most of these seem to fall into that category.